.github/workflows/publish_image_chart.yaml

```diff
@@ -6,6 +6,8 @@ on:
       - master
     tags:
       - 'v*'
+  release:
+    types: [created]
```

Comment:
I don't know how this workflow works when triggered with a release event.
I think it will work as expected. However, it will run twice simultaneously because we usually push a tag by creating a new release.

I suggest to use `startsWith(github.ref, 'refs/tags/')` to run steps only if the tags are pushed.

ref; https://docs.github.com/ja/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#startswith
ref; https://docs.github.com/ja/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context (`github.ref`)

Reply:
Thanks for the comment, addressed by https://github.com/pipe-cd/pipecd/pull/5260/commits/6c7786ed582d252ff0766df48a9b4efa9b538676 👀